web/templates/confirm_dialog/confirm_unstar_all_messages_in_topic.hbs

```diff
@@ -1,6 +1,4 @@
-<p>
-    {{#tr}}
-        Are you sure you want to unstar all messages in <stream-topic></stream-topic>?  This action cannot be undone.
-        {{#*inline "stream-topic"}}{{> ../stream_topic_widget .}}{{/inline}}
-    {{/tr}}
-</p>
+{{#tr}}
+    Are you sure you want to unstar all messages in <stream-topic></stream-topic>?  This action cannot be undone.
+    {{#*inline "stream-topic"}}{{> ../stream_topic_widget .}}{{/inline}}
+{{/tr}}
```

Comment:
Should this one pull `This action cannot be undone.` into a separate string, like you've done with some others, to avoid extra translator effort?